db/migrate/20171010181113_create_answers.rb

```diff
@@ -1,7 +1,7 @@
 class CreateAnswers < ActiveRecord::Migration[5.1]
   def change
     create_table :answers do |t|
-      t.text :text
+      t.text :text, default: ''
```

Comment:
LMAO ok i feel so bad for introducing this bad habit.... so you can make changes to these files UNLESS master has them. When rake db:migrate is run again, these files will not run again. Therefore, the only way to run these migration changes would be to drop the database and recreate, and reseed. This means we would have to do it on production too. You should never ever drop your database on production for migrations.

Reply:
instead, create a new migration that changes the column info